bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdataframe.py

```diff
@@ -388,6 +389,17 @@ def __call__(self, *args):
 
         return res
 
+def _PyFilter(rdf, func, funcargs = {}, filter_name = ""):
+    """
+    Filters the RDF according to the function or string provided.
+    """
+    if isinstance(func, str): # If string argument is passed. Invoke the Orginal Filters.
```

Comment:
Continuing with my comment above, for example this would be `if isinstance(args[0], str)`. Calling it `func` means assuming that it is a function, but it can also be a string.